tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/builtinconstraints/EmailConstraintTest.java

```diff
@@ -49,17 +47,11 @@ public static WebArchive createTestArchive() {
 	@SpecAssertion(section = Sections.BUILTINCONSTRAINTS, id = "v")
 	public void testEmailConstraint() {
 		Validator validator = TestUtil.getValidatorUnderTest();
-		EmailDummyEntity foo = new EmailDummyEntity();
+		EmailDummyEntity foo = new EmailDummyEntity( "invalid" );
 
 		Set<ConstraintViolation<EmailDummyEntity>> constraintViolations = validator.validate( foo );
-		assertNumberOfViolations( constraintViolations, 0 );
-
-		foo.email = "4 8";
-		constraintViolations = validator.validate( foo );
-		assertCorrectConstraintTypes( constraintViolations, Email.class );
-		assertThat( constraintViolations ).containsOnlyPaths(
-				pathWith()
-						.property( "email" )
+		assertThat( constraintViolations ).containsOnlyViolations(
```

Comment:
Ok, yeah, we cannot test for any specific reason of the failure. I.e. we won't be able to tell apart whether the violation is due to the flag being violated or the actual email validation. Seems that's as good as it gets.